aider-helm.el

```diff
@@ -9,7 +9,7 @@
   "History list for `aider-helm-read-string` inputs.")
 
 (defvar aider-helm-read-string-history-file
-  "~/.emacs.d/aider-helm-read-string-history.el"
+  (expand-file-name "aider-helm-read-string-history.el" user-emacs-directory)
```

Comment:
I renamed this file to match the conventions I found for emacs pacakages, to be consistent with the doom one, and updated this line so I could use it, too.

Reply:
happy to revert my changes to this file too, they are unrelated to the PR otherwise

Reply:
oh this change is better than the previous one, cause I don't know this command. Tested and it works as expected.